docs/INSTALLATION.md

```diff
@@ -41,9 +45,11 @@ ng new my-store
 export class AppModule { }
 ```
 
-2. Replace the defaults with your store's url in your `AppModule`
-3. `ng serve`
-3. Visit http://localhost:4200 in your browser
+> Be sure to Replace the defaults with your store's url in your `AppModule`
```

Comment:
`replace`

Reply:
resolved in `d2987766643fb817985c3a9c679558d14a3754f1`